### PR TITLE
Migration

### PR DESCRIPTION
**Pull request by angel-perez-itti**
Pull-request for issue [#17238](https://github.com/slingr-stack/platform/issues/17238) created by @pasaperez-slingr 

## What it does?

Review Google Storage package

## How to test it?

You can use https://package.tarzan.io/dev/builder/#model/packages

Execute the scripts from the readme. 

[Checklist for packages](https://docs.google.com/document/d/1rMDvwVnbCTNuC0Z8z3An3c5UpnvY4S2CUEuJMoLsAAg/edit#heading=h.tn1fsa44no8t)

Remember to use this [checklist](https://docs.google.com/document/d/139rKhIsvpT3yBSB88BBZfy03fD_Sried0bVwIWVaHJk/edit?usp=sharing) for the review.

## Technical notes

There are no technical notes.

## Deployment notes

There are no deployment notes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,147 @@
...
+
+- Authentication via Service Account
+- Direct access to the Google Cloud Storage API
+- Helpers to simplify interactions with Google Cloud Storage in your app
+
+## Configuration
+
+Official documentation: [https://cloud.google.com/storage/docs/json_api](https://cloud.google.com/storage/docs/json_api)
+
+### Registering New Service Account
```
Comment (sebasmoyano) [inline README.md]: The UX standard for documentation is capital case. Please review the entire file.

File: README.md [hunk 2]
```diff
@@ -19,40 +19,40 @@ It provides the following features:
...
 
-#### Private Key
+#### Private key
 
 **Name**: privateKey  
 **Type**: password (text)  
 **Mandatory**: true
 
-# Javascript API
+# Javascript api
```
Comment (sebasmoyano) [inline README.md]: In this case API is an acronym, it remains the same, in capital letters.

File: README.md [hunk 3]
```diff
@@ -19,40 +19,40 @@ It provides the following features:
...
 
-## HTTP Requests
+## Http requests
```
Comment (sebasmoyano) [inline README.md]: It should be HTTP.

File: README.md [hunk 4]
```diff
@@ -135,7 +135,7 @@ for more information about generic requests.
 ## Dependencies
 * HTTP Service
 
-## About SLINGR
+## About slingr
```
Comment (sebasmoyano) [inline README.md]: It should be `Slingr`.

File: README.md [hunk 5]
```diff
@@ -1,16 +1,147 @@
...
+    }
+});
+log(JSON.stringify(response));
+```
+
+### Delete a bucket
+
+```javascript
+let response;
+response = pkg.googlecloudstorage.api.delete('/b/my-bucket');
```
Comment (sebasmoyano) [inline README.md:72]: we don't support camel case declaration of the package? (`googleCloudStorage`)

Comment (pasaperez-work) [reply README.md:72]: We support it, but: 
* we have not done extensive coverage on this
* It depends on the first time you register the package in the developer portal, to use camel case in the namespace we have to register the packages correctly and remove all previous versions and metadata (or register the following new packages with this convention)

Comment (sebasmoyano) [reply README.md:72]: OK, something to keep in mind.

**Comment by sebasmoyano** [comment]
#approve

**Pull request merged**